### ISSUE TITLE
The contents of the list control are not fully displayed, and an error message pops up when you double-click the title bar

### ISSUE THREAD

**Issue by guoiaoang** [AlexanderPro/SmartSystemMenu#140]
The list control content is not displayed completely. When I double-click the title bar, an error message pops up. The title bar does not seem to be draggable. How can I make the content display completely?

![image](https://github.com/user-attachments/assets/0759d442-afa5-4c00-83b3-ab0c9df88555)

**Comment by AlexanderPro** [comment]
Hi,

Do you see a tooltip when your mouse cursor is over the grid cell with a long text? I have reproduced a double click error for the title bar and will fix it in the next version.

**Comment by guoiaoang** [comment]
> Hi,
> 
> Do you see a tooltip when your mouse cursor is over the grid cell with a long text? I have reproduced a double click error for the title bar and will fix it in the next version.

Yes, I can see the tooltip when the mouse hovers. However, the behavior of this control does not seem to be consistent with the standard control. I just tried multiple list controls, and clicking the table header to sort seems to be only partially implemented.

**Comment by AlexanderPro** [comment]
Please, try version [2.27.2](https://github.com/AlexanderPro/SmartSystemMenu/releases/tag/v2.27.2). The list columns are resizable and sortable now.

**Comment by guoiaoang** [comment]
> Please, try version [2.27.2](https://github.com/AlexanderPro/SmartSystemMenu/releases/tag/v2.27.2). The list columns are resizable and sortable now.

After downloading and testing, in the settings of version v2.27.2, no error pop-up window appears when clicking on the header of the list control, and the header columns in the third Page "Menu (Resize) " and the fourth page "Menu (Start Program)" of the settings page have the function of sorting. However, there is no error pop-up window or sorting when clicking on the header of the first page "General" and the second  page "Menu". I remember that the second column of the menu page of the previous version can be sorted. In addition, the width of the page seems a bit narrow.

![image](https://github.com/user-attachments/assets/a71a95cf-b42e-4801-9110-aee179948dd1)

**Comment by AlexanderPro** [comment]
I have decided to make the second column not sortable, as this list has a tree structure with two levels. Are you sure you need this column to be sortable? What font name and scale do you use in your OS?

**Comment by guoiaoang** [comment]
> I have decided to make the second column not sortable, as this list has a tree structure with two levels. Are you sure you need this column to be sortable? What font name and scale do you use in your OS?

By the way, I just started using this software and don't have this need at the moment. But the table on the first tab page can be sorted. The operating system I use is Windows 11 22H2 22621.2861, the resolution is 1680*1050, the zoom is 100%, and the font display is the system default font. I have not adjusted the font, so I don't know how to see the specific font name, but it may be called Microsoft Yahei.

![image](https://github.com/user-attachments/assets/a5f9d7ce-7cc8-407e-a935-7ec702df2665)
